treeherder/webapp/api/bug.py

```diff
@@ -55,14 +55,19 @@ def retrieve(self, request, project, pk=None):
             return Response("Object not found", status=HTTP_404_NOT_FOUND)
 
     def list(self, request, project):
-        job_ids = map(int, request.query_params.getlist('job_id'))
-        if not job_ids:
-            return Response({"message": "At least one job_id is required"},
-                            status=400)
+        try:
+            job_ids = map(int, request.query_params.getlist('job_id'))
+            if not job_ids:
+                return Response({"message": "At least one job_id is required"},
+                                status=400)
 
-        jobs = Job.objects.filter(repository__name=project, id__in=job_ids)
-        bug_job_maps = BugJobMap.objects.filter(job__in=jobs).select_related(
-            'user')
-        serializer = BugJobMapSerializer(bug_job_maps, many=True)
+            jobs = Job.objects.filter(repository__name=project, id__in=job_ids)
+            bug_job_maps = BugJobMap.objects.filter(job__in=jobs).select_related(
+                'user')
+            serializer = BugJobMapSerializer(bug_job_maps, many=True)
 
-        return Response(serializer.data)
+            return Response(serializer.data)
+
+        except ValueError:
```

Comment:
Could you narrow the scope of the try-except, such that it just wraps the `map()` line? This should improve readability of the code (and also prevent catching cases that are unrelated).

Could you also add a test to:
https://github.com/mozilla/treeherder/blob/master/tests/webapp/api/test_bug_job_map_api.py

Many thanks!